### PR TITLE
Change base docker image

### PR DESCRIPTION
**Pull request by emmanuelm41**

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -46,13 +46,14 @@ RUN cargo +1.49.0 install wasm-pack
 RUN cargo +1.49.0 install cargo-tree
```
Comment (jleni) [inline src/Dockerfile:46]: Why 1.49 here?

File: src/Dockerfile [hunk 2]
```diff
@@ -46,13 +46,14 @@ RUN cargo +1.49.0 install wasm-pack
...
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
+RUN apt-get -y install nodejs
+RUN npm install -g yarn
 
+RUN node -v
 RUN yarn --version
-RUN sudo yarn global add n
-RUN sudo n latest -q
+RUN yarn global add n
+RUN n latest -q
```
Comment (jleni) [inline src/Dockerfile]: I think we should go with stable here, not latest

**Review by jleni — CHANGES_REQUESTED** [review]

File: src/Dockerfile (hunk 2 above)
Comment (emmanuelm41) [reply src/Dockerfile]: mmm, i just leave everything the same way it was. Just change the base docker image. Do you want me to change it anyway?

File: src/Dockerfile (hunk 1 above)
Comment (emmanuelm41) [reply src/Dockerfile:46]: Same here. I don't know why. The image is exactly as it was on docker-builder repo, but with another base image

File: src/Dockerfile (hunk 2 above)
Comment (jleni) [reply src/Dockerfile]: Yes, I think it is a good opportunity to review these things.

File: src/Dockerfile (hunk 1 above)
Comment (jleni) [reply src/Dockerfile:46]: Yes, it is a good opportunity to review these things..

Comment (emmanuelm41) [reply src/Dockerfile:46]: Done!

File: src/Dockerfile (hunk 2 above)
Comment (emmanuelm41) [reply src/Dockerfile]: Done!

**Pull request merged**